src/util.c

```diff
@@ -500,4 +500,29 @@ RRStatus parseHashSlots(char * slots, char * string)
 exit:
     RedisModule_Free(string);
     return ret;
-}
\ No newline at end of file
+}
+
+ClientState *ClientStateGet(RedisRaftCtx *rr, RedisModuleCtx * ctx)
```

Comment:
```suggestion
ClientState *ClientStateGet(RedisRaftCtx *rr, RedisModuleCtx *ctx)
```
There were two misaligned pointer in this line, the previous suggestion had both. Consider accepting changes via github, you don't miss the changes and it's just one click. I find it easier than manual commit.